frontend/public/components/utils/kebab.tsx

```diff
@@ -329,6 +330,15 @@ const kebabFactory: KebabFactory = {
       }),
     accessReview: asAccessReview(kind, obj, 'patch'),
   }),
+  ClonePVC: (kind, obj) => ({
+    label: 'Clone PVC',
+    callback: () =>
+      clonePVCModal({
+        kind,
+        resource: obj,
+      }),
+    accessReview: asAccessReview(kind, obj, 'patch'),
```

Comment:
I am a bit confused about the verb here shouldn't it be `create` as we are creating a new entity?

Reply:
good one, it should be create.